Automattic/woocommerce-payments
# Fix pay for order flow missing user session email

Fixes #7708 

#### Changes proposed in this Pull Request
This PR check that the email is set in the post global, because `sanitize_email` will return an empty string if the email is not set.


#### Testing instructions
1. As a guest, create a pay-for-order order in incognito and keep the browser open
2. Find the pay-for-order order from WC->Orders and copy the page to the above browser and a different tab
3. You should be able to see the pay-for-order page without login
4. Checkout with WooPay

-------------------


**Post merge**
